lib/src/components/NxCheckbox/NxCheckbox.scss

```diff
@@ -21,24 +25,47 @@ $-width: 16px;
     @include nx-focus-border-helpers.radio-checkbox-focus-border($-height, $-width);
 
     border-radius: 4px;
-    left: -5px;
-    top: -5px;
+    left: -4px;
```

Comment:
```suggestion
    left: calc(var(--nx-focus-outline-offset) + var(--nx-focus-outline-width));
```

Reply:
The same applies to the `top` on the line right below this

Reply:
> The same applies to the `top` on the line right below this

Fixed: 6c8b03cdcf54133c67d8788fbc853d0685364067